.eslintrc

```diff
@@ -196,7 +198,6 @@
     "space-before-function-paren": 0,
     "space-in-parens": [1, "never"],
     "space-infix-ops": 1,
-    "space-return-throw-case": 1,
```

Comment:
Actually, upon looking up this rule I don't know why I was breaking it, going to look into it:
http://eslint.org/docs/rules/space-return-throw-case